salt/modules/win_appx.py

```diff
@@ -0,0 +1,361 @@
+"""
+Manage provisioned apps
+=======================
+
+Provisioned apps are part of the image and are installed for every user the
+first time the user logs on. Provisioned apps are also updated and sometimes
+reinstalled when the system is updated.
+
+Apps removed with this module will remove the app for all users and deprovision
+the app. Deprovisioned apps will neither be installed for new users nor will
+they be upgraded.
+
+An app removed with this module can only be re-provisioned on the machine, but
+it can't be re-installed for all users. Also, once a package has been
+deprovisioned, the only way to reinstall it is to download the package. This is
+difficult. I've outlined the steps below:
+
+1. Obtain the Microsoft Store URL for the app:
+    - Open the page for the app in the Microsoft Store
+    - Click the share button and copy the URL
+
+2. Look up the packages on https://store.rg-adguard.net/:
+    - Ensure ``URL (link)`` is selected in the first dropdown
+    - Paste the URL in the search field
+    - Ensure Retail is selected in the 2nd dropdown
+    - Click the checkmark button
+
+This should give you a list of URLs for the package and all dependencies for all
```

Comment:
```suggestion
This should give a list of URLs for the package and all dependencies for all
```

Reply:
I changed it to `This should return a list...`